helpers/utils

```diff
@@ -139,24 +142,28 @@ ynh_setup_source() {
     mkdir -p /var/cache/yunohost/download/${YNH_APP_ID}/
     src_filename="/var/cache/yunohost/download/${YNH_APP_ID}/${src_filename}"
 
-    if test -e "$local_src"; then
-        cp $local_src $src_filename
+    if [ "$src_format" = "docker" ]; then
+        src_plateform="${src_plateform:-"linux/$YNH_ARCH"}"
     else
-        [ -n "$src_url" ] || ynh_die "Couldn't parse SOURCE_URL from $src_file_path ?"
-
-        # NB. we have to declare the var as local first,
-        # otherwise 'local foo=$(false) || echo 'pwet'" does'nt work
-        # because local always return 0 ...
-        local out
-        # Timeout option is here to enforce the timeout on dns query and tcp connect (c.f. man wget)
-        out=$(wget --tries 3 --no-dns-cache --timeout 900 --no-verbose --output-document=$src_filename $src_url 2>&1) \
-            || ynh_die --message="$out"
+        if test -e "$local_src"; then
```

Comment:
Ideally we can use `elif` instead of `else ....... if` to avoid an entire indentation level ;P

Reply:
done, I let you check it